functions/function/aggregator-function/src/main/java/org/springframework/cloud/fn/aggregator/AggregatorFunctionConfiguration.java

```diff
@@ -83,7 +85,8 @@ public AggregatorFactoryBean aggregator(
 			ObjectProvider<ReleaseStrategy> releaseStrategy,
 			ObjectProvider<MessageGroupProcessor> messageGroupProcessor,
 			ObjectProvider<MessageGroupStore> messageStore,
-			@Qualifier("outputChannel") MessageChannel outputChannel) {
+			@Qualifier("outputChannel") MessageChannel outputChannel,
+			@Nullable ComponentCustomizer<AggregatorFactoryBean> aggregatorCustomizer) {
```

Comment:
For consistency should we use an `ObjectProvider` in this particular case since all of the other deps are using that rather than `@Nullable`?

Reply:
I just didn’t know back in days that `@Nullable` will work. Or it just didn’t work in those old Spring versions… 😅

Reply:
Makes sense. Should we keep it consistent in this case and move to ObjectProvider to "fit in" w/ the other params?